em-client/src/models.rs

```diff
@@ -929,7 +929,7 @@ pub struct ApplicationConfigPort {
     /// 
     #[serde(rename = "application")]
     #[serde(skip_serializing_if="Option::is_none")]
-    pub application: Option<serde_json::Value>,
+    pub application: Option<String>,
```

Comment:
As discussed offline, this is not needed if you check the hash before the json is parsed and returned